src/components/page-header/navigation/navigation-item/navigation-item.component.ts

```diff
@@ -91,10 +91,14 @@ export class PageHeaderNavigationItemComponent implements AfterViewInit, OnDestr
         this._pageHeaderService.select(this.item);
     }
 
-    @HostListener('keydown', ['$event'])
-    onKeydown(event: KeyboardEvent): void {
+    @HostListener('keydown', ['$event', '$event.target'])
+    onKeydown(event: KeyboardEvent, target: HTMLElement): void {
         if (event.keyCode === LEFT_ARROW || event.keyCode === RIGHT_ARROW) {
             this._navigationService.onKeydown(event);
         }
+
+        if (event.keyCode === SPACE && target.getAttribute('href')) {
```

Comment:
Is this `href` check necessary? This event handler is only attached to an element which has a non-empty `routerLink` defined, so I would assume this would always evaluate to true.